src/frequenz/dispatch/_actor_runner.py

```diff
@@ -0,0 +1,187 @@
+# License: All rights reserved
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Helper class to manage actors based on dispatches."""
+
+import logging
+from dataclasses import dataclass
+from typing import Any
+
+from frequenz.channels import Receiver, Sender
+from frequenz.client.dispatch.types import ComponentSelector
+from frequenz.sdk.actor import Actor
+
+from ._dispatch import Dispatch, RunningState
+
+_logger = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class DispatchConfigurationEvent:
```

Comment:
I would call this `DispatchUpdate`. I think calling this configuration is a bit confusing because we use *configuration* for something else, and also I think even should be "created", "updated", etc. For me this class only holds a *dispatch update*.